cfn-resources/util/util.go

```diff
@@ -143,6 +149,75 @@ func NewMongoDBClient(req handler.Request, profileName *string) (*mongodbatlas.C
 	return mongodbClient, nil
 }
 
+// NewAtlasClient func for creating atlas-go-sdk and mongodb-atlas-go client
+func NewAtlasClient(req *handler.Request, profileName *string) (*MongoDBClient, *handler.ProgressEvent) {
```

Comment:
wasn't this added in another PR?

Reply:
This is addressed by pulling latest from main